docs/articles/documentation/guides/basic-guides/run-tests.md

```diff
@@ -497,19 +498,19 @@ In this case, tests are distributed across four Safari instances and the same te
 
 > If an uncaught error or unhandled promise rejection occurs on the server during test execution, all tests running concurrently will fail.
 
-When you run tests on [remote devices](../../reference/command-line-interface.md#remote-browsers),
-create connections for each instance of each browser you test against. When using
-the command line interface, specify this number after the `remote:` keyword. In API, create
-a [browser connection](../../reference/testcafe-api/browserconnection/README.md) for each instance.
+### Use Concurrency on Remote Devices
 
-On a remote device, invoke all the required instances manually. The total number of instances
-should divide by the concurrency parameter `n`. Otherwise, an exception will be thrown.
+When you run tests on [remote devices](../../reference/command-line-interface.md#remote-browsers), create browser connections for each instance of each browser you test against. The total number of instances is *c\*n*, where `c` is the concurrency factor and `n` is the number of browsers.
+
+In the command line interface, specify the number of browser instances after the `remote:` keyword.  For example, use `-c 2 remote:6` to run tests in Chrome, Firefox, and Edge, with two instances of each browser. If you use the API, create a [browser connection](../../reference/testcafe-api/browserconnection/README.md) for each instance in code.
+
+Launch all the required browser instances manually on a remote device. The total number of instances should divide by the concurrency factor `c`. Otherwise, an exception is thrown.
```

Comment:
`The total number of instances should divide by the concurrency factor `c`.`

Do you mean that the total number of instances are "divided by" the concurrency factor `c`? Please clarify.

Reply:
I would say (and @DIRECTcut also mentioned it in [his review](https://github.com/DevExpress/testcafe/pull/5804#discussion_r544361910)) "should be divisible by", but I remember getting a correction to "should divide by".

Does "should be divisible by" make sense?

Reply:
"**x** should be divisible by **y**" means that when you divide **x** by **y**, the remainder is zero. For example, "4 is divisible by 2" is correct because when you divide 4 by 2, the remainder is zero.

Is this what you're trying to say here?

Reply:
Yes, that's what I mean. Thank you, I'll use "should be divisible" then, right?

Reply:
Yes, you can use "should be divisible by" in this context.